libraries/my-controls/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "my-controls",
+  "version": "1.0.0",
+  "description": "An example library project",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "scripts": {
+    "build": "node_modules/.bin/my-build"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "my-toolchain": "^1.0.0",
+    "typescript": "^3.0.3"
```

Comment:
>typescript [](start = 5, length = 10)

is this necessary?

Reply:
Not technically, but I think it's useful to illustrate a dependency used by all three projects

---
In reply to: [216846025](https://github.com/Microsoft/rush-example/pull/1#discussion_r216846025) [](ancestors = 216846025)